2010-05-04  Andreas Schwab  <schwab@redhat.com>

	* SUPPORTED (SUPPORTED-LOCALES): Fix name of tt_RU.UTF-8@iqtelif
	locale.
 
diff --git a/localedata/SUPPORTED b/localedata/SUPPORTED
index a1ebcac..d665961 100644
--- a/localedata/SUPPORTED
+++ b/localedata/SUPPORTED
@@ -386,7 +386,7 @@ tr_TR.UTF-8/UTF-8 \
 tr_TR/ISO-8859-9 \
 ts_ZA/UTF-8 \
 tt_RU.UTF-8/UTF-8 \
-tt_RU@iqtelif.UTF-8/UTF-8 \
+tt_RU.UTF-8@iqtelif/UTF-8 \
 ug_CN/UTF-8 \
 uk_UA.UTF-8/UTF-8 \
 uk_UA/KOI8-U \
